packages/account/package.json

```diff
@@ -23,7 +22,7 @@
     "tsc": "ethereumjs-config-tsc",
     "lint": "ethereumjs-config-lint",
     "lint:fix": "ethereumjs-config-lint-fix",
-    "test": "npx tape -r ts-node/register test/*.spec.ts"
+    "test": "nyc tape -r ts-node/register test/*.spec.ts"
```

Comment:
I wonder if I would rather prefer to keep a separate `coverage` command here (or is this difficult due to changed conditions on the CI setup?). I am using `npm run test` locally all the time for test runs and I can imagine that I would find it rather distracting to have coverage reported all the time along. So for my taste this is binding things unnecessarily together.

Reply:
Agreed!